File: Python/1470. Shuffle the Array.py
nums = list(map(int, input().split()))
n = int(input())

newArray = []
x = True
xCount = 0
yCount = n
for i in range(2*n):
    if x:
        newArray.append(nums[xCount])
        xCount += 1
        x = False
    else:
        newArray.append(nums[yCount])
        yCount += 1
        x = True
print(newArray)

File: Javascript/1470. Shuffle the Array.js
var shuffle = (nums, n) => {
    const temp = [];
    for (let i = 0; i < n; i++) {
        temp.push(nums[i]);
        temp.push(nums[i + n]);
    }
    return temp;
};

console.log(shuffle([2, 5, 1, 3, 4, 7], 3));
